monitor/dbt_project.py

```diff
@@ -0,0 +1,15 @@
+import os
```

Comment:
I think I would rename the file to dbt_project_utils or something like that because now we have dbt_project directory and dbt_project file which is a bit confusing 🙂